src/Monitors/IMUDownlink.cpp

```diff
@@ -40,14 +39,14 @@ void IMUDownlink::execute()
             prev_deployed = sfr::mission::deployed;
         }
 
-        // added to or statment for the IMU booting
-        if (sfr::mission::current_mode->get_id() == sfr::mission::bootImu->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::mandatoryBurns->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::regularBurns->get_id()) {
+        // Determine when it should sample_gyro
+        if (sfr::mission::current_mode->get_id() == sfr::mission::bootImu->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::bootCamera->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::mandatoryBurns->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::regularBurns->get_id() || sfr::mission::current_mode->get_id() == sfr::mission::photo->get_id()) {
             sfr::imu::sample_gyro = true;
-        }
+        } else
```

Comment:
This logic should not be included here- it should be in the transition_to methods for these mission modes.